src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
```

Comment:
The constructor of the Herbivore class should not accept the 'health' parameter as it should always be 100 by default according to the task description. The 'health' parameter should be removed, and the 'super' call should only pass 'name'.